src/style.css

```diff
@@ -0,0 +1,62 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
```

Comment:
You should add separate @font-face for normal font-weight, and for bold font-weight and use needed styles for inputs and for placeholders